pylsp/python_lsp.py

```diff
@@ -109,7 +109,7 @@ def start_ws_lang_server(port, check_parent_process, handler_class):
         import websockets
     except ImportError as e:
         raise ImportError(
-            "websocket modules missing. Please run pip install 'python-lsp-server[websockets]"
+            "websocket modules missing. Please run 'pip install python-lsp-server[websockets]'"
```

Comment:
It's the `python-lsp-server[websockets]` part that needs quotes to work in a shell.

Reply:
Interesting, in my terminal it's not necessary, so I wanted to use quotes to mark a part to copy.
What about adding a colon and quotes as you suggest (it works for me):

```
websocket modules missing. Please run: pip install 'python-lsp-server[websockets]'
```

Reply:
sounds good to me